README.md

```diff
@@ -37,42 +37,81 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
-    - [Dependencies](#dependencies)
-    - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
+    - [Dockerfile](#dockerfile)
+    - [Non-pythonic dependencies](#non-pythonic-dependencies)
+    - [Pythonic dependencies](#pythonic-dependencies)
+      - [Test installation](#test-installation)
     - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
   - [Computational performances](#computational-performances)
   - [Package development](#package-development)
 
 ## Installation
 
-The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
 
-### Dependencies
+### Dockerfile
 
-Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.10 installed. The following dependency installation instructions are updated as of 14/09/2023, but in case of issues during installation always refer to the official documentation which is linked below:
+In order to try out the package without worrying about your OS and without the need of installing all the required dependencies, we created a `Dockerfile` that can be used for taking care of everything in a suitable container. After having cloned the repository and installed [Docker](https://docs.docker.com/engine/install/), run the following commands from the root of the repository.
+
+Build the Docker image:
+```bash
+docker build -t deeprank2 .
+```
```

Comment:
Did not work for me:
> ERROR: failed to solve: process "/bin/sh -c apt update -y &&   apt install unzip -y &&   apt install -y gcc &&   wget https://github.com/PDB-REDO/dssp/releases/download/v4.4.0/mkdssp-4.4.0-linux-x64 &&   mv mkdssp-4.4.0-linux-x64 /usr/local/bin/mkdssp &&   chmod a+x /usr/local/bin/mkdssp &&   conda env create -f /home/environment.yml &&   wget https://zenodo.org/records/8349335/files/data_raw.zip &&   unzip data_raw.zip -d data_raw &&   mv data_raw /home/deeprank2/tutorials" did not complete successfully: exit code: 1

Scrolling further up, I noticed:
> 168.0 mv: cannot move 'data_raw' to '/home/deeprank2/tutorials/data_raw': Directory not empty

After discussing with @dsmits, we realized that the problem is that I had already downloaded the raw data into my local folder (with same name) and that that is why the docker installation cannot add the same files. Another thing I noticed is that because I also have the "processed" data folder in my local repo, the docker image will also use those, and that is multiple GB of data, which we don't necessarily want/need in the docker image.

In any case, there is some conflict between having downloaded the data already and docker trying to download it that needs to be resolved.

Reply:
If you follow the instructions, it is written that you should clone the repo from scratch and then build the image (so no data will be in tutorials/). Also, having downloaded the data already is not something that a user would have when trying out the package for the first time.

Reply:
I still feel that it's confusing and error prone. If someone doesn't follow the steps to the letter (maybe already started looking at the tutorials) and downloads the data beforehand or if (like me), their build will fail. 
Either we should be more explicit about making sure those folders are empty, or create a workaround for this scenario. Personally, I don't think it should have to be required to re-clone the repo. If anyone already has the repo, the assumption will be that by just pulling the latest updates, the build should work and it's kind of annoying to have to clone a fresh one.

Reply:
It should be fixed now